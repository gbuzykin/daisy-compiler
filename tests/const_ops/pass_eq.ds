#if 99871u32 != 99871i32
#error invalid result
#endif
#if 47i8 != 47i32
#error invalid result
#endif
#if 47u8 != 47i32
#error invalid result
#endif
#if 47i8 != 47u32
#error invalid result
#endif
#if 47u8 != 47u32
#error invalid result
#endif
#if 4294867425u32 != -99871i32
#error invalid result
#endif
#if -47i8 != -47i32
#error invalid result
#endif
#if -47i8 != 4294967249u32
#error invalid result
#endif
