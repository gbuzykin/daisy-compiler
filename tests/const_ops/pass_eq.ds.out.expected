./const_ops/pass_eq.ds: info: warnings 0, errors 0
